### DISCUSSION TITLE
Attempted to generate a new E3SM case using "--res f19_g16 --compset WCYCL20TR," but encountered failure in the process

### DISCUSSION THREAD

**Discussion by jingyizhuo** [E3SM-Project/E3SM#5877] [E3SM model help]
Hi dear E3SM developer,

I attempted to generate a new E3SM case using "--res f19_g16 --compset WCYCL20TR," but encountered failure in the process. Would you give some suggestion on this?

the error info I got after the command of "./case.setup" is as follows:
Create namelist for component eam
   Calling /global/u2/t/tcjinny/code/E3SM/components/eam//cime_config/buildnml
ERROR: Command: '/global/u2/t/tcjinny/code/E3SM/components/eam/bld/build-namelist -infile /global/homes/t/tcjinny/cases/E3SM_v2/f19_g17_BHIST_cntl/Buildconf/eamconf/cesm_namelist -csmdata /global/cfs/cdirs/e3sm/inputdata -ignore_ic_year -use_case 20TR_eam_CMIP6 -inputdata /global/homes/t/tcjinny/cases/E3SM_v2/f19_g17_BHIST_cntl/Buildconf/eam.input_data_list -ntasks 64.0 -namelist " &atmexp  /" ' failed with error 'CAM build-namelist - ERROR: No default value found for ncdata' from dir '/global/homes/t/tcjinny/cases/E3SM_v2/f19_g17_BHIST_cntl/Buildconf/eamconf'

**Comment by mt5555** [comment]
That's a CESM resolution - running that grid would require the CAM-FV lat/lon dycore, and the POP ocean model, neither of which are supported in E3SM.

**Reply by jingyizhuo** [reply]
Hi Mark,

Thanks a lot for your response and explanation!!  So I modified the resolution and compset parameters to "--res ne30pg2_EC30to60E2r2 --compset WCYCL20TR" within E3SM. The new case still has a issue during running, seemingly linked to the absence of forcing data. 

The concluding section of the atm.log is provided below for your reference.

(GETFIL): attempting to find local file Solar_1850-2299_input4MIPS_c20181106.nc
 (GETFIL): using /global/cfs/cdirs/e3sm/inputdata/atm/cam/solar/Solar_1850-2299_input4MIPS_c20181106.nc
 Opened existing file /global/cfs/cdirs/e3sm/inputdata/atm/cam/solar/Solar_1850-2299_input4MIPS_c20181106.nc          29
 solar_data_init: data file = /global/cfs/cdirs/e3sm/inputdata/atm/cam/solar/Solar_1850-2299_input4MIPS_c20181106.nc
 solar_data_init: has_ref_spectrum F
solar_data_advance: not able to find data for : 0001-01-01-00000
 ERROR: solar_data_advance: failed to read data from /global/cfs/cdirs/e3sm/inputdata/atm/cam/solar/Solar_1850-2299_input4MIPS_c20181106.nc

I have also executed the "check_input_data --download" to check for input data, but unfortunately, the same issue persists. Do you have any further suggestions or insights to offer regarding this matter?  I am new to the E3SM, thank you once again for your assistance.

**Reply by jonbob** [reply]
I believe you're trying to run a 20th-century configuration starting from year 0001, and the forcing data is not on the specified files. Try setting:
````
./xmlchange RUN_STARTDATE=1850-01-01
````
and see if that fixes the problem

**Reply by jingyizhuo** [reply]
You are so right!!! This is where the problem lies. Thanks for helping out.
